build-tools/packages/build-cli/src/commands/merge/branches.ts

```diff
@@ -219,7 +217,7 @@ export default class MergeBranch extends BaseCommand<typeof MergeBranch> {
 
 		if (prWillConflict === false) {
 			await this.gitRepo.gitClient
-				.merge([flags.target, "-m", prTitle])
+				.merge([`${this.remote}/${flags.target}`, "-m", prTitle])
```

Comment:
This is confusing to me again. Why do you want to merge with the remote branch instead of the local branch?

Reply:
If you're trying to ensure you merge with the latest, you would have to do a `git fetch` to update the remote refs anyway, so merging with the remote ref doesn't really save you anything.